### DISCUSSION TITLE
Overriding default 401 response

### DISCUSSION THREAD

**Discussion by securityncompliance** [solidusio/solidus#5107] [Troubleshooting]
When I modify the bearer token (swap/add couple letters) to simulate the 401, it works as intended. I am a bit annoyed by the response that says:

```
{"error":"No verification key available"}
```

Is there any way we can customize this response? Here is what I have tried:

1) Tried adding rails middleware that catches the JWT::DecodeError then if caught sends a different error (didn't work)
2) I tried catching JWT::DecodeError in the method that executes authorized_user! in my base api controller (same result)
3) I tried some crazy stuff with warden failure app and few other futile attempts 

Am I over-thinking this, can anyone offer a different suggestion or approach? (Yes I have tried chatGPT it had no clue as well)

**Comment by kennyadsl** [comment]
Hello, @securityncompliance! Which error message are you expecting exactly? Also, are you using https://github.com/solidusio-contrib/solidus_jwt?
